corehq/apps/userreports/es/adapter.py

```diff
@@ -0,0 +1,44 @@
+from elasticsearch import NotFoundError
+from corehq.apps.userreports.util import get_table_name
+from corehq.apps.userreports.adapter import IndicatorAdapter
+from corehq.elastic import get_es_new
+
+
+class IndicatorESAdapter(IndicatorAdapter):
+
+    def __init__(self, config):
+        super(IndicatorESAdapter, self).__init__(config)
+        self.es = get_es_new()
+        self.table_name = get_table_name(config.domain, config.table_id)
+
+    def rebuild_table(self):
+        self.drop_table()
+        self.es.indices.create(index=self.table_name)
```

Comment:
have we done any research into how many ES indices are fine to have in a particular server before you start running into issues?